client/my-sites/earn/components/add-edit-plan-modal/index.tsx

```diff
@@ -66,36 +86,40 @@ const currencyList = Object.keys( STRIPE_MINIMUM_CURRENCY_AMOUNT ).map( ( code )
  * @param {string} connectedAccountDefaultCurrency - Default currency of the current account
  * @returns {number} Minimum transaction amount for given currency.
  */
-function minimumCurrencyTransactionAmount( currency, connectedAccountDefaultCurrency ) {
+function minimumCurrencyTransactionAmount(
+	currency: string,
+	connectedAccountDefaultCurrency: string
+): number {
 	if ( connectedAccountDefaultCurrency === currency.toUpperCase() ) {
-		return STRIPE_MINIMUM_CURRENCY_AMOUNT[ currency ];
+		return Number( STRIPE_MINIMUM_CURRENCY_AMOUNT[ currency ] );
```

Comment:
Was surprised by this `Number()` cast — why do we need it since values are already numbers? In case the key in array doesn't exist? In that case we should fail in some other way than convert.

Reply:
Yes, you're right. This isn't needed here. I've removed it.